### PR TITLE
More changes

### PR DESCRIPTION
**Pull request by Ramit110**
Made a big chunk of changes.
Mostly to fix errors inside decisions.

### CODE REVIEW

File: scopes.cwt [hunk 1]
```diff
@@ -1900,4 +1900,9 @@ alias[effect:random_owned_bloodline] = {
...
 	}
 	alias_name[effect] = alias_match_left[effect]
-}
\ No newline at end of file
+}
+
+## push_scope = character
+## scope = character
+### 
+alias[trigger:faction_claimant] = single_alias_right[trigger_clause]
```
Comment (tboby) [inline scopes.cwt]: Is this a "link", e.g. a scope change that works anywhere?
If yes, move to links.cwt

**Comment by tboby** [comment]
One comment, otherwise LGTM

File: decision/decision_pre_triggers_and_alias.cwt [hunk 2]
```diff
@@ -79,7 +86,16 @@ single_alias[filtered_decisions] = {
     ### Factors for an AI character to take the decision (1 = 100% chance)
     ai_will_do = single_alias_right[decision_ai_will_do_clause]
 
-    # TODO: ai_accceptance
+    ## cardinality = 0..1
+    ## replace_scope = { this = character root = character from = character }
+    ai_acceptance = {
+        factor = int
+        yes_threshold = int
+        show_maybe_for_human = bool
```
Comment (UristMcDorf) [inline decision/decision_pre_triggers_and_alias.cwt:95]: Defaults? Ranges for the above? Should keep a #TODO comment above if you're unwilling to do them now :)

File: scopes.cwt (hunk 1 above)
Comment (Ramit110) [reply scopes.cwt]: Afaik it is so it's been moved.

**Pull request merged**